lib/src/child-writer/index.js

```diff
@@ -56,20 +58,46 @@ const EXIT_CODES = require('../exit-codes');
 exports.write = (image, drive, options) => {
   const emitter = new EventEmitter();
 
-  utils.getTemporaryLogFilePath().then((logFile) => {
-    const argv = utils.getCLIWriterArguments({
-      entryPoint: rendererUtils.getApplicationEntryPoint(),
-      logFile: logFile,
-      image: image,
-      device: drive.device,
-      validateWriteOnSuccess: options.validateWriteOnSuccess,
-      unmountOnSuccess: options.unmountOnSuccess
-    });
+  const argv = utils.getCLIWriterArguments({
+    entryPoint: rendererUtils.getApplicationEntryPoint(),
+    image: image,
+    device: drive.device,
+    validateWriteOnSuccess: options.validateWriteOnSuccess,
+    unmountOnSuccess: options.unmountOnSuccess
+  });
+
+  // There might be multiple Etcher instances running at
+  // the same time, therefore we must ensure each IPC
+  // server/client has a randomly generated name.
```

Comment:
I haven't considered PIDs, which is way more elegant, but as @lurch pointed out, it'd allow other potentially malicious processes to inject information to the GUI.

Even if a malicious process sends stuff to the IPC server, there is no possible harm to be done to the user. The IPC server is just used to display progress information, so I guess we're safe to go with PIDs.